lib/improver/wind_downscaling.py

```diff
@@ -914,6 +924,7 @@ def process(self, input_cube):
         if not isinstance(input_cube, iris.cube.Cube):
             msg = "wind input is not a cube, but {}"
             raise TypeError(msg.format(type(input_cube)))
+        input_cube.data = input_cube.data.astype(np.float32)
```

Comment:
Again, recommend "if cube.dtype == np.float64" before doing this, particularly if the cube has several dimensions.  Also an inline comment that we are doing this for optimisation purposes.